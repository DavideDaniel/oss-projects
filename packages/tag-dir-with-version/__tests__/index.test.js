import * as tagModule from '../index.mjs';

describe('tag-dir-with-version', () => {
  it('should match its snapshot', () => {
    expect(tagModule).toMatchInlineSnapshot();
  });
});
