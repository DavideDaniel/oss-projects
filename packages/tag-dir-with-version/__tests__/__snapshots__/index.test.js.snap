// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`tag-dir-with-version should match its snapshot 1`] = `
Object {
  "tagDirWithVersion": [Function],
}
`;
